met-api/src/met_api/resources/engagement_metadata.py

```diff
@@ -131,11 +146,16 @@ class EngagementMetadataById(Resource):
 
     @staticmethod
     @cross_origin(origins=allowedorigins())
-    @require_role(VIEW_ENGAGEMENT_ROLES)
+    @_jwt.requires_auth
```

Comment:
Are we sure about requiring authorization for this route? What if we're pulling engagement metadata on the frontend for public users?